projects/duplicate-char-counter/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Duplicate Character Counting in Every Language
+layout: default
+date: 2020-10-04
+last-modified: 2020-10-04
+featured-image: <name of featured image file in assets folder>
+tags: [<a list of tags>]
+authors:
+  - GoodbyeBlues
+---
+
+An overview of the duplicate characters counting problem.
+
+## Goal
+
+Although a one line solution for this problem is feasible, just as huge brute force solutions are, we should avoid both of these approaches. Instead, our goal should be to find a fine line between both, where the code is not bogged down but is largely readable and maintainable
+
+## Requirements
+
+The code should return the count of all duplicate, case-insensitive, alphanumeric characters, that occur more than once in the given string. 
+
+## Examples
```

Comment:
```suggestion
## Testing
```

This section looks good! For consistency, can we turn it into a table? It would have three columns: description, input, and output.